### PR TITLE
8 footer

### PR DESCRIPTION
**Pull request by rawand1998**
here we add the footer to home page and made style for it.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,15 +5,18 @@
...
 </head>
 <body>
     <footer>
-        <img src="" alt="">
-        <div class="">
-            <button>Contact Us</button>
-            <button>About Us</button>
-            <button>Join Us</button>
+        <div class="image-footer">
+          <img src="logo2.jpg" alt="" />
```
Comment (Mu7ammadAbed) [inline index.html]: Add a real descriptive image alternative

File: index.html [hunk 2]
```diff
@@ -5,15 +5,18 @@
...
         </div>
-    </footer>
+        <div class="footer-button">
+          <button>Contact Us</button>
```
Comment (Mu7ammadAbed) [inline index.html]: If you want a button to redirect you to another page or section you can use anchor tag but don't use a button for redirection, if you want the shape and style of the button you can style the anchor to look like a button by css but not using a button.

File: styles/home.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
+footer {
```
Comment (Mu7ammadAbed) [inline styles/home.css]: If you want to select an html element to style it you should add a class name to it and select it by class name for multiple reasons, specificity is one of them.

**Review by Mu7ammadAbed — CHANGES_REQUESTED** [review]

File: index.html [hunk 4]
```diff
@@ -5,15 +5,18 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
+    <link rel="stylesheet" href="./styles/home.css">
 </head>
 <body>
     <footer>
-        <img src="" alt="">
```
Comment (umarsalah) [inline index.html]: Add a real image src, we add a new logo photo.

**Review by umarsalah — CHANGES_REQUESTED** [review]

File: styles/home.css [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+  width: 80%;
+  justify-content: space-between;
+  bottom: 0;
+  height: 150px;
+  position: fixed;
+  bottom: 0;
+  margin-left: 7%;
+}
+footer img {
```
Comment (Mu7ammadAbed) [inline styles/home.css]: The same as before here, just add a class name to the image and select it

**Review by umarsalah — APPROVED** [review]

**Review by muathkhatib — APPROVED** [review]

**Review by Mu7ammadAbed — APPROVED** [review]

**Pull request merged**